ppa-mg-copy-packages.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/python
```

Comment:
This is still untested stuff! Will do some more work on it and add proper documentation if something can be automated (it's for copying packages in the PPA)